elements/lisk-transactions/src/utils/index.ts

```diff
@@ -13,6 +13,7 @@
  *
  */
 export * from './address';
+export * from './constructor';
```

Comment:
constructor is reserved word, it should use other name.
`safe_get_ids` might be better? depends on the function names